pkg/innerring/processors/netmap/process_epoch.go

```diff
@@ -45,12 +45,16 @@ func (np *Processor) processNewEpoch(ev netmapEvent.NewEpoch) {
 		return
 	}
 
+	estimationEpoch := epoch - 1
+
 	prm := cntClient.StartEstimationPrm{}
 
-	prm.SetEpoch(epoch - 1)
+	prm.SetEpoch(estimationEpoch)
 	prm.SetHash(ev.TxHash())
 
 	if epoch > 0 { // estimates are invalid in genesis epoch
+		np.log.Debug("start estimation collection", zap.Uint64("epoch", estimationEpoch))
```

Comment:
Is it once per epoch? We can do `Info` then.

Reply:
> Is it once per epoch?

yes. well, that is debug to me, but do not mind, changed

Reply:
It was completely unused, so a pretty easy choice here.

Reply:
We have other per-epoch things like
```
neofs-ir[255165]: 2023-08-09T12:58:17.156Z        info        settlement/calls.go:20        new audit settlement event        {"epoch": 7916}
neofs-ir[255165]: 2023-08-09T12:58:17.156Z        info        settlement/handlers.go:14        process audit settlements        {"epoch": 7916}
neofs-ir[255165]: 2023-08-09T12:58:17.156Z        info        audit/calculate.go:61        calculate audit settlements        {"current epoch": 7916}
neofs-ir[255165]: 2023-08-09T12:58:17.156Z        info        governance/handlers.go:33        new event        {"type": "sync"}
neofs-ir[255165]: 2023-08-09T12:58:17.174Z        info        settlement/handlers.go:18        audit processing finished        {"epoch": 7916}
```
So it should be OK.